python/pyspark/sql/streaming.py

```diff
@@ -315,9 +315,9 @@ def load(self, path=None, format=None, schema=None, **options):
         :param schema: optional :class:`pyspark.sql.types.StructType` for the input schema.
         :param options: all other string options
 
-        >>> json_sdf = spark.readStream.format("json")\
-                                       .schema(sdf_schema)\
-                                       .load(tempfile.mkdtemp())
+        >>> json_sdf = spark.readStream.format("json") \\
```

Comment:
I see, does the \\ vs \ make the difference here? are there possibly other instances of this?